sympy/combinatorics/pc_groups.py

```diff
@@ -3,30 +3,16 @@
 from sympy.combinatorics.perm_groups import PermutationGroup
 from sympy.printing.defaults import DefaultPrinting
 
-class PolycyclicGroup(Basic):
+class PolycyclicGroup(DefaultPrinting):
 
     is_group = True
     is_solvable = True
 
-    def __init__(self, pcgs_):
-        self.perm_group = PermutationGroup(pcgs_)
-        self.pc_series = self._pc_series()
-        self.pcgs = self._compute_pcgs()
+    def __init__(self, pc_sequence, pc_series):
+        self.pc_series = pc_series
```

Comment:
I think that the groups in the polycyclic series should be computed from the generator sequence: `<x0, ..., xn>, <x1, ...xn>, ..., <xn>, 1`. That would guarantee that generator the sequence and the polycyclic series are compatible. (There are many alternatives for both the sequence and the series.)